engine/engine/src/engine.cpp

```diff
@@ -590,6 +632,11 @@ namespace dmEngine
 #endif
         }
 
+        if (LoadSslKeys(argc, argv, content_root, dmConfigFile::GetString(engine->m_Config, "network.ssl_certificates", "")))
```

Comment:
Since both LoadSSLKeys and GetProjectFile are called from `Init()` and both use the resources path, I suggest you call GetResourcesPath first.

```
char resource_path[DMPATH_MAX_PATH];
dmSys::GetResourcesPath(argc, argv, resource_path, sizeof(resource_path));
...
GetProjectFile(..., resources_path);
...
char ssl_keys_path[DMPATH_MAX_PATH];
dmStrlCpy(ssl_keys_path, resource_path, sizeof(ssl_keys_path));
dmPath::Concat(ssl_keys_path, dmConfigFile::GetString(engine->m_Config, "network.ssl_certificates", ""));
LoadSSlKeys(ssl_keys_path);
```